Больше деталей

Allender45 opened the pull request:
---
:mortar_board: [Больше деталей](https://up.htmlacademy.ru/javascript/28/user/1105189/tasks/9)

Allender45 committed:
Задание: 4.16. Больше деталей

senchkim commented on js/main.js:
```diff
@@ -0,0 +1,60 @@
+let id = 1;
```
Давай мы удалим все `console.log` - пока ты разрабатываешь то пользуйся нет проблем, но комитить их не нужно!

senchkim commented on js/main.js:
```diff
@@ -0,0 +1,60 @@
+let id = 1;
+let commentId = 1;
+
+const descriptions = [
```
Все константы которые содержат статичный текст сделаем с большой буквы!
https://up.htmlacademy.ru/javascript/28/criteries#b7
`descriptions` => `DESCRIPTIONS`

senchkim commented on js/main.js:
```diff
@@ -0,0 +1,60 @@
+  'Последнее'
+];
+
+const messages = [
```
`messages` => `MESSAGES`

senchkim commented on js/main.js:
```diff
@@ -0,0 +1,60 @@
+  'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!'
+];
+
+const names = [
```
`names` => `NAMES`

senchkim commented on js/main.js:
```diff
@@ -0,0 +1,60 @@
+  'Милледи'
+];
+
+function getRandomInteger(min, max) {
```
https://up.htmlacademy.ru/javascript/28/criteries#d5
давай пож-ста все функции сделаем в едином стиле - либо стрелочные либо через ключевое слово `function` 👍

senchkim commented on js/main.js:
```diff
@@ -0,0 +1,60 @@
+};
+
+
+const array = () => {
```
https://up.htmlacademy.ru/javascript/28/criteries#b6
`array` => `getArray`

senchkim commented on js/main.js:
```diff
@@ -0,0 +1,60 @@
+
+const getObjects = Array.from({length: 3}, array);
+
+console.log(getObjects);
```
В остальном все хорошо 💯

senchkim commented:
и еще пока не забыл - перед тем как открывать пулл-реквест запускай команду
`npm run lint` - она подскажет тебе ошибки по линтеру)

Allender45 committed:
Работа с замечаниями.
Вывод в консоль оставил только для того, чтобы тебе было проще проверять. Вообще код, который ничего не делает, это тоже ошибка. Хотя он мог вызываться в другом файле конечно.
Линтер встроен в вебшторм. Но почему-то на название констант и вид функций он не ругался. Постараюсь в будущем соответствовать.
